.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "MSAL"
```

Comment:
What is this part used for?

Reply:
This prevents VSCode from teating "MSAL" like a misspelled word.